--- a/net/minecraft/client/searchtree/SearchRegistry.java
+++ b/net/minecraft/client/searchtree/SearchRegistry.java
@@ -27,7 +27,7 @@
    }
 
    private <T> SearchRegistry.TreeEntry<T> m_225361_(SearchRegistry.Key<T> p_225362_) {
-      SearchRegistry.TreeEntry<T> treeentry = this.f_119944_.get(p_225362_);
+      SearchRegistry.TreeEntry<T> treeentry = (SearchRegistry.TreeEntry<T>)this.f_119944_.get(p_225362_);
       if (treeentry == null) {
          throw new IllegalStateException("Tree builder not registered");
       } else {
@@ -58,7 +58,7 @@
       }
 
       void m_218383_(List<T> p_218384_) {
-         this.f_214440_ = this.f_215486_.apply((T)p_218384_);
+         this.f_214440_ = this.f_215486_.apply(p_218384_);
          this.f_214440_.m_217089_();
       }
 
