--- a/net/minecraft/client/resources/model/ModelBakery.java
+++ b/net/minecraft/client/resources/model/ModelBakery.java
@@ -354,7 +354,7 @@
                      try {
                         Resource resource = p_223332_.m_214158_();
 
-                        Pair pair2;
+                        Pair<String, BlockModelDefinition> pair2;
                         try {
                            InputStream inputstream = resource.m_6679_();
 
