--- a/net/minecraft/commands/synchronization/ArgumentTypeInfos.java
+++ b/net/minecraft/commands/synchronization/ArgumentTypeInfos.java
@@ -110,8 +110,8 @@
       m_223443_(p_223442_, "entity_summon", EntitySummonArgument.class, SingletonArgumentInfo.m_223508_(EntitySummonArgument::m_93335_));
       m_223443_(p_223442_, "dimension", DimensionArgument.class, SingletonArgumentInfo.m_223508_(DimensionArgument::m_88805_));
       m_223443_(p_223442_, "time", TimeArgument.class, SingletonArgumentInfo.m_223508_(TimeArgument::m_113037_));
-      m_223443_(p_223442_, "resource_or_tag", m_223452_(ResourceOrTagLocationArgument.class), new ResourceOrTagLocationArgument.Info());
-      m_223443_(p_223442_, "resource", m_223452_(ResourceKeyArgument.class), new ResourceKeyArgument.Info());
+      m_223443_(p_223442_, "resource_or_tag", m_223452_(ResourceOrTagLocationArgument.class), new ResourceOrTagLocationArgument.Info<Object>());
+      m_223443_(p_223442_, "resource", m_223452_(ResourceKeyArgument.class), new ResourceKeyArgument.Info<Object>());
       if (SharedConstants.f_136183_) {
          m_223443_(p_223442_, "test_argument", TestFunctionArgument.class, SingletonArgumentInfo.m_223508_(TestFunctionArgument::m_128088_));
          m_223443_(p_223442_, "test_class", TestClassNameArgument.class, SingletonArgumentInfo.m_223508_(TestClassNameArgument::m_127917_));
@@ -121,7 +121,7 @@
    }
 
    private static <T extends ArgumentType<?>> Class<T> m_223452_(Class<? super T> p_223453_) {
-      return p_223453_;
+      return (Class<T>)p_223453_;
    }
 
    public static boolean m_223448_(Class<?> p_223449_) {
@@ -133,7 +133,7 @@
       if (argumenttypeinfo == null) {
          throw new IllegalArgumentException("Unrecognized argument type %s (%s)".formatted(p_223440_, p_223440_.getClass()));
       } else {
-         return argumenttypeinfo;
+         return (ArgumentTypeInfo<A, ?>)argumenttypeinfo;
       }
    }
 
