--- a/net/minecraft/client/OptionInstance.java
+++ b/net/minecraft/client/OptionInstance.java
@@ -107,7 +107,7 @@
    }
 
    public AbstractWidget m_233985_(Options p_233986_, int p_233987_, int p_233988_, int p_233989_) {
-      OptionInstance.TooltipSupplier<T> tooltipsupplier = this.f_214612_.apply((T)Minecraft.m_91087_());
+      OptionInstance.TooltipSupplier<T> tooltipsupplier = this.f_214612_.apply(Minecraft.m_91087_());
       return this.f_214445_.m_219922_(tooltipsupplier, p_233986_, p_233987_, p_233988_, p_233989_).apply(this);
    }
 
@@ -264,7 +264,7 @@
       default <R> OptionInstance.SliderableValueSet<R> m_217989_(final IntFunction<? extends R> p_217990_, final ToIntFunction<? super R> p_217991_) {
          return new OptionInstance.SliderableValueSet<R>() {
             public Optional<R> m_216842_(R p_217619_) {
-               return IntRangeBase.this.m_216842_((T)Integer.valueOf(p_217991_.applyAsInt(p_217619_))).map(p_217990_::apply);
+               return IntRangeBase.this.m_216842_(Integer.valueOf(p_217991_.applyAsInt(p_217619_))).map(p_217990_::apply);
             }
 
             public double m_218050_(R p_233996_) {
