--- a/net/minecraft/util/datafix/fixes/OptionsKeyLwjgl3Fix.java
+++ b/net/minecraft/util/datafix/fixes/OptionsKeyLwjgl3Fix.java
@@ -163,7 +163,7 @@
                      return Pair.of(p_145580_.getKey(), p_145580_.getValue());
                   }
                }).collect(Collectors.toMap(Pair::getFirst, Pair::getSecond)));
-            }).result().orElse(p_145575_);
+            }).result().orElse((com.mojang.serialization.Dynamic) p_145575_);
          });
       });
    }
