--- a/net/minecraft/client/gui/screens/worldselection/WorldGenSettingsComponent.java
+++ b/net/minecraft/client/gui/screens/worldselection/WorldGenSettingsComponent.java
@@ -186,7 +186,7 @@
    }
 
    private static Component m_220448_(Holder<WorldPreset> p_220449_) {
-      return p_220449_.m_203543_().map((p_235774_) -> {
+      return p_220449_.m_203543_().<Component>map((p_235774_) -> {
          return Component.m_228094_(p_235774_.m_135782_().m_220238_("generator"));
       }).orElse(f_101382_);
    }
