--- a/net/minecraft/client/OptionInstance.java
+++ b/net/minecraft/client/OptionInstance.java
@@ -275,7 +275,7 @@
       default <R> OptionInstance.SliderableValueSet<R> m_231657_(final IntFunction<? extends R> p_231658_, final ToIntFunction<? super R> p_231659_) {
          return new OptionInstance.SliderableValueSet<R>() {
             public Optional<R> m_214064_(R p_231674_) {
-               return IntRangeBase.this.m_214064_((T)Integer.valueOf(p_231659_.applyAsInt(p_231674_))).map(p_231658_::apply);
+               return IntRangeBase.this.m_214064_(Integer.valueOf(p_231659_.applyAsInt(p_231674_))).map(p_231658_::apply);
             }
 
             public double m_213640_(R p_231678_) {
