--- a/net/minecraft/server/packs/VanillaPackResources.java
+++ b/net/minecraft/server/packs/VanillaPackResources.java
@@ -167,7 +167,7 @@
       try {
          stream.filter((p_225620_) -> {
             return !p_225620_.endsWith(".mcmeta") && Files.isRegularFile(p_225620_);
-         }).mapMulti((p_229186_, p_229187_) -> {
+         }).<ResourceLocation>mapMulti((p_229186_, p_229187_) -> {
             String s = path.relativize(p_229186_).toString().replaceAll("\\\\", "/");
             ResourceLocation resourcelocation = ResourceLocation.m_227477_(p_229818_, s);
             if (resourcelocation == null) {
