--- a/net/minecraft/network/syncher/EntityDataSerializer.java
+++ b/net/minecraft/network/syncher/EntityDataSerializer.java
@@ -18,11 +18,11 @@
    static <T> EntityDataSerializer<T> m_224556_(final FriendlyByteBuf.Writer<T> p_224557_, final FriendlyByteBuf.Reader<T> p_224558_) {
       return new EntityDataSerializer.ForValueType<T>() {
          public void m_6856_(FriendlyByteBuf p_227274_, T p_227275_) {
-            p_224557_.accept((T)p_227274_, p_227275_);
+            p_224557_.accept(p_227274_, p_227275_);
          }
 
          public T m_6709_(FriendlyByteBuf p_226262_) {
-            return p_224558_.apply((T)p_226262_);
+            return p_224558_.apply(p_226262_);
          }
       };
    }
