--- a/net/minecraft/world/entity/monster/warden/WardenAi.java
+++ b/net/minecraft/world/entity/monster/warden/WardenAi.java
@@ -94,15 +94,15 @@
    }
 
    private static void m_229026_(Brain<Warden> p_229027_) {
-      p_229027_.m_21891_(Activity.f_37979_, 10, ImmutableList.of(new SetRoarTarget<>(Warden::m_229966_), new TryToSniff(), new RunOne(ImmutableMap.of(MemoryModuleType.f_214128_, MemoryStatus.VALUE_ABSENT), ImmutableList.of(Pair.of(new RandomStroll(0.5F), 2), Pair.of(new DoNothing(30, 60), 1)))));
+      p_229027_.m_21891_(Activity.f_37979_, 10, ImmutableList.of(new SetRoarTarget<>(Warden::m_229966_), new TryToSniff(), new RunOne<>(ImmutableMap.of(MemoryModuleType.f_214128_, MemoryStatus.VALUE_ABSENT), ImmutableList.of(Pair.of(new RandomStroll(0.5F), 2), Pair.of(new DoNothing(30, 60), 1)))));
    }
 
    private static void m_229604_(Brain<Warden> p_229605_) {
-      p_229605_.m_21895_(Activity.f_214988_, 5, ImmutableList.of(new SetRoarTarget<>(Warden::m_229966_), new GoToTargetLocation(MemoryModuleType.f_214439_, 2, 0.7F)), MemoryModuleType.f_214439_);
+      p_229605_.m_21895_(Activity.f_214988_, 5, ImmutableList.of(new SetRoarTarget<>(Warden::m_229966_), new GoToTargetLocation<>(MemoryModuleType.f_214439_, 2, 0.7F)), MemoryModuleType.f_214439_);
    }
 
    private static void m_235197_(Brain<Warden> p_235198_) {
-      p_235198_.m_21895_(Activity.f_215092_, 5, ImmutableList.of(new SetRoarTarget<>(Warden::m_229966_), new Sniffing(f_214715_)), MemoryModuleType.f_214128_);
+      p_235198_.m_21895_(Activity.f_215092_, 5, ImmutableList.of(new SetRoarTarget<>(Warden::m_229966_), new Sniffing<>(f_214715_)), MemoryModuleType.f_214128_);
    }
 
    private static void m_223186_(Brain<Warden> p_223187_) {
