--- a/net/minecraft/world/entity/ai/behavior/warden/SetWardenLookTarget.java
+++ b/net/minecraft/world/entity/ai/behavior/warden/SetWardenLookTarget.java
@@ -13,7 +13,7 @@
 public class SetWardenLookTarget {
    public static BehaviorControl<LivingEntity> m_257897_() {
       return BehaviorBuilder.m_258034_((p_258946_) -> {
-         return p_258946_.<MemoryAccessor, MemoryAccessor, MemoryAccessor, MemoryAccessor>group(p_258946_.m_257492_(MemoryModuleType.f_26371_), p_258946_.m_257492_(MemoryModuleType.f_217783_), p_258946_.m_257492_(MemoryModuleType.f_217782_), p_258946_.m_258080_(MemoryModuleType.f_26372_)).apply(p_258946_, (p_258942_, p_258943_, p_258944_, p_258945_) -> {
+         return p_258946_.group(p_258946_.m_257492_(MemoryModuleType.f_26371_), p_258946_.m_257492_(MemoryModuleType.f_217783_), p_258946_.m_257492_(MemoryModuleType.f_217782_), p_258946_.m_258080_(MemoryModuleType.f_26372_)).apply(p_258946_, (p_258942_, p_258943_, p_258944_, p_258945_) -> {
             return (p_258936_, p_258937_, p_258938_) -> {
                Optional<BlockPos> optional = p_258946_.<LivingEntity>m_257828_(p_258944_).map(Entity::m_20183_).or(() -> {
                   return p_258946_.m_257828_(p_258943_);
