--- a/net/minecraft/world/entity/ai/behavior/warden/TryToSniff.java
+++ b/net/minecraft/world/entity/ai/behavior/warden/TryToSniff.java
@@ -15,7 +15,7 @@
 
    public static BehaviorControl<LivingEntity> m_257812_() {
       return BehaviorBuilder.m_258034_((p_259979_) -> {
-         return p_259979_.<MemoryAccessor, MemoryAccessor, MemoryAccessor, MemoryAccessor, MemoryAccessor>group(p_259979_.m_257492_(MemoryModuleType.f_217785_), p_259979_.m_257492_(MemoryModuleType.f_26370_), p_259979_.m_258080_(MemoryModuleType.f_217772_), p_259979_.m_257495_(MemoryModuleType.f_148194_), p_259979_.m_258080_(MemoryModuleType.f_217783_)).apply(p_259979_, (p_260219_, p_260252_, p_260090_, p_259577_, p_260020_) -> {
+         return p_259979_.group(p_259979_.m_257492_(MemoryModuleType.f_217785_), p_259979_.m_257492_(MemoryModuleType.f_26370_), p_259979_.m_258080_(MemoryModuleType.f_217772_), p_259979_.m_257495_(MemoryModuleType.f_148194_), p_259979_.m_258080_(MemoryModuleType.f_217783_)).apply(p_259979_, (p_260219_, p_260252_, p_260090_, p_259577_, p_260020_) -> {
             return (p_259589_, p_259399_, p_260081_) -> {
                p_260219_.m_257512_(Unit.INSTANCE);
                p_260090_.m_257465_(Unit.INSTANCE, (long)f_217735_.m_214085_(p_259589_.m_213780_()));
