--- a/net/minecraft/world/entity/monster/piglin/RememberIfHoglinWasKilled.java
+++ b/net/minecraft/world/entity/monster/piglin/RememberIfHoglinWasKilled.java
@@ -10,7 +10,7 @@
 public class RememberIfHoglinWasKilled {
    public static BehaviorControl<LivingEntity> m_257745_() {
       return BehaviorBuilder.m_258034_((p_260168_) -> {
-         return p_260168_.<MemoryAccessor, MemoryAccessor>group(p_260168_.m_257495_(MemoryModuleType.f_26372_), p_260168_.m_257492_(MemoryModuleType.f_26340_)).apply(p_260168_, (p_259214_, p_260031_) -> {
+         return p_260168_.group(p_260168_.m_257495_(MemoryModuleType.f_26372_), p_260168_.m_257492_(MemoryModuleType.f_26340_)).apply(p_260168_, (p_259214_, p_260031_) -> {
             return (p_260324_, p_259380_, p_260313_) -> {
                LivingEntity livingentity = p_260168_.m_258051_(p_259214_);
                if (livingentity.m_6095_() == EntityType.f_20456_ && livingentity.m_21224_()) {
