--- a/net/minecraft/world/level/levelgen/presets/WorldPreset.java
+++ b/net/minecraft/world/level/levelgen/presets/WorldPreset.java
@@ -17,7 +17,7 @@
 import net.minecraft.world.level.levelgen.WorldDimensions;
 
 public class WorldPreset {
-   public static final Codec<WorldPreset> f_226414_ = RecordCodecBuilder.create((p_259011_) -> {
+   public static final Codec<WorldPreset> f_226414_ = RecordCodecBuilder.<WorldPreset>create((p_259011_) -> {
       return p_259011_.group(Codec.unboundedMap(ResourceKey.m_195966_(Registries.f_256862_), LevelStem.f_63970_).fieldOf("dimensions").forGetter((p_226430_) -> {
          return p_226430_.f_226416_;
       })).apply(p_259011_, WorldPreset::new);
