--- a/net/minecraft/world/entity/ai/behavior/BackUpIfTooClose.java
+++ b/net/minecraft/world/entity/ai/behavior/BackUpIfTooClose.java
@@ -11,7 +11,7 @@
 public class BackUpIfTooClose {
    public static OneShot<Mob> m_257698_(int p_259782_, float p_259071_) {
       return BehaviorBuilder.m_258034_((p_260278_) -> {
-         return p_260278_.<MemoryAccessor, MemoryAccessor, MemoryAccessor, MemoryAccessor>group(p_260278_.m_258080_(MemoryModuleType.f_26370_), p_260278_.m_257492_(MemoryModuleType.f_26371_), p_260278_.m_257495_(MemoryModuleType.f_26372_), p_260278_.m_257495_(MemoryModuleType.f_148205_)).apply(p_260278_, (p_260206_, p_259953_, p_259993_, p_259209_) -> {
+         return p_260278_.group(p_260278_.m_258080_(MemoryModuleType.f_26370_), p_260278_.m_257492_(MemoryModuleType.f_26371_), p_260278_.m_257495_(MemoryModuleType.f_26372_), p_260278_.m_257495_(MemoryModuleType.f_148205_)).apply(p_260278_, (p_260206_, p_259953_, p_259993_, p_259209_) -> {
             return (p_259617_, p_260038_, p_259374_) -> {
                LivingEntity livingentity = p_260278_.m_258051_(p_259993_);
                if (livingentity.m_19950_(p_260038_, (double)p_259782_) && p_260278_.<NearestVisibleLivingEntities>m_258051_(p_259209_).m_186107_(livingentity)) {
