--- a/net/minecraft/world/entity/ai/behavior/InteractWithDoor.java
+++ b/net/minecraft/world/entity/ai/behavior/InteractWithDoor.java
@@ -33,7 +33,7 @@
       MutableObject<Node> mutableobject = new MutableObject<>((Node)null);
       MutableInt mutableint = new MutableInt(0);
       return BehaviorBuilder.m_258034_((p_258474_) -> {
-         return p_258474_.<MemoryAccessor, MemoryAccessor, MemoryAccessor>group(p_258474_.m_257495_(MemoryModuleType.f_26377_), p_258474_.m_257492_(MemoryModuleType.f_26379_), p_258474_.m_257492_(MemoryModuleType.f_148204_)).apply(p_258474_, (p_258460_, p_258461_, p_258462_) -> {
+         return p_258474_.group(p_258474_.m_257495_(MemoryModuleType.f_26377_), p_258474_.m_257492_(MemoryModuleType.f_26379_), p_258474_.m_257492_(MemoryModuleType.f_148204_)).apply(p_258474_, (p_258460_, p_258461_, p_258462_) -> {
             return (p_258469_, p_258470_, p_258471_) -> {
                Path path = p_258474_.m_258051_(p_258460_);
                Optional<Set<GlobalPos>> optional = p_258474_.m_257828_(p_258461_);
