--- a/net/minecraft/world/entity/ai/behavior/MeleeAttack.java
+++ b/net/minecraft/world/entity/ai/behavior/MeleeAttack.java
@@ -13,7 +13,7 @@
 public class MeleeAttack {
    public static OneShot<Mob> m_257733_(int p_259758_) {
       return BehaviorBuilder.m_258034_((p_258526_) -> {
-         return p_258526_.<MemoryAccessor, MemoryAccessor, MemoryAccessor, MemoryAccessor>group(p_258526_.m_257492_(MemoryModuleType.f_26371_), p_258526_.m_257495_(MemoryModuleType.f_26372_), p_258526_.m_258080_(MemoryModuleType.f_26373_), p_258526_.m_257495_(MemoryModuleType.f_148205_)).apply(p_258526_, (p_258529_, p_258530_, p_258531_, p_258532_) -> {
+         return p_258526_.group(p_258526_.m_257492_(MemoryModuleType.f_26371_), p_258526_.m_257495_(MemoryModuleType.f_26372_), p_258526_.m_258080_(MemoryModuleType.f_26373_), p_258526_.m_257495_(MemoryModuleType.f_148205_)).apply(p_258526_, (p_258529_, p_258530_, p_258531_, p_258532_) -> {
             return (p_258539_, p_258540_, p_258541_) -> {
                LivingEntity livingentity = p_258526_.m_258051_(p_258530_);
                if (!m_23527_(p_258540_) && p_258540_.m_217066_(livingentity) && p_258526_.<NearestVisibleLivingEntities>m_258051_(p_258532_).m_186107_(livingentity)) {
