--- a/net/minecraft/world/entity/ai/behavior/ResetProfession.java
+++ b/net/minecraft/world/entity/ai/behavior/ResetProfession.java
@@ -10,7 +10,7 @@
 public class ResetProfession {
    public static BehaviorControl<Villager> m_257593_() {
       return BehaviorBuilder.m_258034_((p_259684_) -> {
-         return p_259684_.<MemoryAccessor>group(p_259684_.m_258080_(MemoryModuleType.f_26360_)).apply(p_259684_, (p_260035_) -> {
+         return p_259684_.group(p_259684_.m_258080_(MemoryModuleType.f_26360_)).apply(p_259684_, (p_260035_) -> {
             return (p_260244_, p_260084_, p_259597_) -> {
                VillagerData villagerdata = p_260084_.m_7141_();
                if (villagerdata.m_35571_() != VillagerProfession.f_35585_ && villagerdata.m_35571_() != VillagerProfession.f_35596_ && p_260084_.m_7809_() == 0 && villagerdata.m_35576_() <= 1) {
