--- a/net/minecraft/world/entity/ai/behavior/SetWalkTargetFromAttackTargetIfTargetOutOfReach.java
+++ b/net/minecraft/world/entity/ai/behavior/SetWalkTargetFromAttackTargetIfTargetOutOfReach.java
@@ -21,7 +21,7 @@
 
    public static BehaviorControl<Mob> m_257648_(Function<LivingEntity, Float> p_259507_) {
       return BehaviorBuilder.m_258034_((p_258687_) -> {
-         return p_258687_.<MemoryAccessor, MemoryAccessor, MemoryAccessor, MemoryAccessor>group(p_258687_.m_257492_(MemoryModuleType.f_26370_), p_258687_.m_257492_(MemoryModuleType.f_26371_), p_258687_.m_257495_(MemoryModuleType.f_26372_), p_258687_.m_257492_(MemoryModuleType.f_148205_)).apply(p_258687_, (p_258699_, p_258700_, p_258701_, p_258702_) -> {
+         return p_258687_.group(p_258687_.m_257492_(MemoryModuleType.f_26370_), p_258687_.m_257492_(MemoryModuleType.f_26371_), p_258687_.m_257495_(MemoryModuleType.f_26372_), p_258687_.m_257492_(MemoryModuleType.f_148205_)).apply(p_258687_, (p_258699_, p_258700_, p_258701_, p_258702_) -> {
             return (p_258694_, p_258695_, p_258696_) -> {
                LivingEntity livingentity = p_258687_.m_258051_(p_258701_);
                Optional<NearestVisibleLivingEntities> optional = p_258687_.m_257828_(p_258702_);
