--- a/net/minecraft/world/entity/ai/behavior/SetWalkTargetFromBlockMemory.java
+++ b/net/minecraft/world/entity/ai/behavior/SetWalkTargetFromBlockMemory.java
@@ -14,7 +14,7 @@
 public class SetWalkTargetFromBlockMemory {
    public static OneShot<Villager> m_257972_(MemoryModuleType<GlobalPos> p_259685_, float p_259842_, int p_259530_, int p_260360_, int p_259504_) {
       return BehaviorBuilder.m_258034_((p_258717_) -> {
-         return p_258717_.<MemoryAccessor, MemoryAccessor, MemoryAccessor>group(p_258717_.m_257492_(MemoryModuleType.f_26326_), p_258717_.m_258080_(MemoryModuleType.f_26370_), p_258717_.m_257495_(p_259685_)).apply(p_258717_, (p_258709_, p_258710_, p_258711_) -> {
+         return p_258717_.group(p_258717_.m_257492_(MemoryModuleType.f_26326_), p_258717_.m_258080_(MemoryModuleType.f_26370_), p_258717_.m_257495_(p_259685_)).apply(p_258717_, (p_258709_, p_258710_, p_258711_) -> {
             return (p_258727_, p_258728_, p_258729_) -> {
                GlobalPos globalpos = p_258717_.m_258051_(p_258711_);
                Optional<Long> optional = p_258717_.m_257828_(p_258709_);
