--- a/net/minecraft/world/entity/ai/behavior/StartCelebratingIfTargetDead.java
+++ b/net/minecraft/world/entity/ai/behavior/StartCelebratingIfTargetDead.java
@@ -11,7 +11,7 @@
 public class StartCelebratingIfTargetDead {
    public static BehaviorControl<LivingEntity> m_257496_(int p_259238_, BiPredicate<LivingEntity, LivingEntity> p_259102_) {
       return BehaviorBuilder.m_258034_((p_259600_) -> {
-         return p_259600_.<MemoryAccessor, MemoryAccessor, MemoryAccessor, MemoryAccessor>group(p_259600_.m_257495_(MemoryModuleType.f_26372_), p_259600_.m_257492_(MemoryModuleType.f_26334_), p_259600_.m_258080_(MemoryModuleType.f_26341_), p_259600_.m_257492_(MemoryModuleType.f_26342_)).apply(p_259600_, (p_259049_, p_259067_, p_259031_, p_259141_) -> {
+         return p_259600_.group(p_259600_.m_257495_(MemoryModuleType.f_26372_), p_259600_.m_257492_(MemoryModuleType.f_26334_), p_259600_.m_258080_(MemoryModuleType.f_26341_), p_259600_.m_257492_(MemoryModuleType.f_26342_)).apply(p_259600_, (p_259049_, p_259067_, p_259031_, p_259141_) -> {
             return (p_259956_, p_259611_, p_259619_) -> {
                LivingEntity livingentity = p_259600_.m_258051_(p_259049_);
                if (!livingentity.m_21224_()) {
