--- a/net/minecraft/world/entity/ai/behavior/StayCloseToTarget.java
+++ b/net/minecraft/world/entity/ai/behavior/StayCloseToTarget.java
@@ -11,7 +11,7 @@
 public class StayCloseToTarget {
    public static BehaviorControl<LivingEntity> m_257901_(Function<LivingEntity, Optional<PositionTracker>> p_259150_, int p_260151_, int p_260087_, float p_259454_) {
       return BehaviorBuilder.m_258034_((p_259273_) -> {
-         return p_259273_.<MemoryAccessor, MemoryAccessor>group(p_259273_.m_257492_(MemoryModuleType.f_26371_), p_259273_.m_258080_(MemoryModuleType.f_26370_)).apply(p_259273_, (p_259832_, p_259483_) -> {
+         return p_259273_.group(p_259273_.m_257492_(MemoryModuleType.f_26371_), p_259273_.m_258080_(MemoryModuleType.f_26370_)).apply(p_259273_, (p_259832_, p_259483_) -> {
             return (p_260054_, p_260069_, p_259517_) -> {
                Optional<PositionTracker> optional = p_259150_.apply(p_260069_);
                if (optional.isEmpty()) {
