--- a/net/minecraft/world/entity/ai/behavior/TryFindWater.java
+++ b/net/minecraft/world/entity/ai/behavior/TryFindWater.java
@@ -15,7 +15,7 @@
    public static BehaviorControl<PathfinderMob> m_257908_(int p_259298_, float p_259140_) {
       MutableLong mutablelong = new MutableLong(0L);
       return BehaviorBuilder.m_258034_((p_260101_) -> {
-         return p_260101_.<MemoryAccessor, MemoryAccessor, MemoryAccessor>group(p_260101_.m_258080_(MemoryModuleType.f_26372_), p_260101_.m_258080_(MemoryModuleType.f_26370_), p_260101_.m_257492_(MemoryModuleType.f_26371_)).apply(p_260101_, (p_259124_, p_259692_, p_259819_) -> {
+         return p_260101_.group(p_260101_.m_258080_(MemoryModuleType.f_26372_), p_260101_.m_258080_(MemoryModuleType.f_26370_), p_260101_.m_257492_(MemoryModuleType.f_26371_)).apply(p_260101_, (p_259124_, p_259692_, p_259819_) -> {
             return (p_260228_, p_259212_, p_260041_) -> {
                if (p_260228_.m_6425_(p_259212_.m_20183_()).m_205070_(FluidTags.f_13131_)) {
                   return false;
