--- a/net/minecraft/world/entity/ai/behavior/YieldJobSite.java
+++ b/net/minecraft/world/entity/ai/behavior/YieldJobSite.java
@@ -19,7 +19,7 @@
 public class YieldJobSite {
    public static BehaviorControl<Villager> m_257788_(float p_259768_) {
       return BehaviorBuilder.m_258034_((p_258916_) -> {
-         return p_258916_.<MemoryAccessor, MemoryAccessor, MemoryAccessor, MemoryAccessor, MemoryAccessor>group(p_258916_.m_257495_(MemoryModuleType.f_26361_), p_258916_.m_258080_(MemoryModuleType.f_26360_), p_258916_.m_257495_(MemoryModuleType.f_148204_), p_258916_.m_257492_(MemoryModuleType.f_26370_), p_258916_.m_257492_(MemoryModuleType.f_26371_)).apply(p_258916_, (p_258901_, p_258902_, p_258903_, p_258904_, p_258905_) -> {
+         return p_258916_.group(p_258916_.m_257495_(MemoryModuleType.f_26361_), p_258916_.m_258080_(MemoryModuleType.f_26360_), p_258916_.m_257495_(MemoryModuleType.f_148204_), p_258916_.m_257492_(MemoryModuleType.f_26370_), p_258916_.m_257492_(MemoryModuleType.f_26371_)).apply(p_258916_, (p_258901_, p_258902_, p_258903_, p_258904_, p_258905_) -> {
             return (p_258912_, p_258913_, p_258914_) -> {
                if (p_258913_.m_6162_()) {
                   return false;
