--- a/net/minecraft/commands/synchronization/ArgumentTypeInfos.java
+++ b/net/minecraft/commands/synchronization/ArgumentTypeInfos.java
@@ -112,8 +112,8 @@
       m_222934_(p_227505_, "entity_summon", EntitySummonArgument.class, SingletonArgumentInfo.m_223454_(EntitySummonArgument::m_93335_));
       m_222934_(p_227505_, "dimension", DimensionArgument.class, SingletonArgumentInfo.m_223454_(DimensionArgument::m_88805_));
       m_222934_(p_227505_, "time", TimeArgument.class, SingletonArgumentInfo.m_223454_(TimeArgument::m_113037_));
-      m_222934_(p_227505_, "resource_or_tag", m_219276_(ResourceOrTagLocationArgument.class), new ResourceOrTagLocationArgument.Info());
-      m_222934_(p_227505_, "resource", m_219276_(ResourceKeyArgument.class), new ResourceKeyArgument.Info());
+      m_222934_(p_227505_, "resource_or_tag", m_219276_(ResourceOrTagLocationArgument.class), new ResourceOrTagLocationArgument.Info<Object>());
+      m_222934_(p_227505_, "resource", m_219276_(ResourceKeyArgument.class), new ResourceKeyArgument.Info<Object>());
       m_222934_(p_227505_, "template_mirror", TemplateMirrorArgument.class, SingletonArgumentInfo.m_223454_(TemplateMirrorArgument::m_229622_));
       m_222934_(p_227505_, "template_rotation", TemplateRotationArgument.class, SingletonArgumentInfo.m_223454_(TemplateRotationArgument::m_234283_));
       if (SharedConstants.f_136183_) {
@@ -125,7 +125,7 @@
    }
 
    private static <T extends ArgumentType<?>> Class<T> m_219276_(Class<? super T> p_219277_) {
-      return p_219277_;
+      return (Class<T>)p_219277_;
    }
 
    public static boolean m_217143_(Class<?> p_217144_) {
@@ -137,7 +137,7 @@
       if (argumenttypeinfo == null) {
          throw new IllegalArgumentException("Unrecognized argument type %s (%s)".formatted(p_228602_, p_228602_.getClass()));
       } else {
-         return argumenttypeinfo;
+         return (ArgumentTypeInfo<A, ?>)argumenttypeinfo;
       }
    }
 
