--- a/net/minecraft/client/searchtree/SearchRegistry.java
+++ b/net/minecraft/client/searchtree/SearchRegistry.java
@@ -30,7 +30,7 @@
    }
 
    private <T> SearchRegistry.TreeEntry<T> m_223399_(SearchRegistry.Key<T> p_223400_) {
-      SearchRegistry.TreeEntry<T> treeentry = this.f_119944_.get(p_223400_);
+      SearchRegistry.TreeEntry<T> treeentry = (SearchRegistry.TreeEntry<T>)this.f_119944_.get(p_223400_);
       if (treeentry == null) {
          throw new IllegalStateException("Tree builder not registered");
       } else {
@@ -64,7 +64,7 @@
       }
 
       void m_221125_(List<T> p_221126_) {
-         this.f_214305_ = this.f_214356_.apply((T)p_221126_);
+         this.f_214305_ = this.f_214356_.apply(p_221126_);
          this.f_214305_.m_224190_();
       }
 
