--- a/net/minecraft/client/gui/screens/PresetFlatWorldScreen.java
+++ b/net/minecraft/client/gui/screens/PresetFlatWorldScreen.java
@@ -261,7 +261,7 @@
 
          public Entry(Holder<FlatLevelGeneratorPreset> p_220902_) {
             this.f_169357_ = p_220902_.m_203334_();
-            this.f_215715_ = p_220902_.m_203543_().map((p_228576_) -> {
+            this.f_215715_ = p_220902_.m_203543_().<Component>map((p_228576_) -> {
                return Component.m_225505_(p_228576_.m_135782_().m_228450_("flat_world_preset"));
             }).orElse(PresetFlatWorldScreen.f_215924_);
          }
