--- a/net/minecraft/resources/RegistryResourceAccess.java
+++ b/net/minecraft/resources/RegistryResourceAccess.java
@@ -151,7 +151,7 @@
          return this.f_195913_.entrySet().stream().filter((p_227559_) -> {
             return p_227559_.getKey().m_135783_(p_218783_);
          }).collect(Collectors.toMap((p_228758_) -> {
-            return p_228758_.getKey();
+            return (ResourceKey<E>)p_228758_.getKey();
          }, (p_226281_) -> {
             return p_226281_.getValue()::m_225260_;
          }));
